test/does-user-win-test.js

```diff
@@ -0,0 +1,4 @@
+import { doesUserWin } from 'get-random-throw.js';
```

Comment:
github is throwing an error since you need to use your imports, looks like you still need testing, but so far looks great!